<launch>
  <group ns="gimbal">
    <node name="controller"
          pkg="storm32_gimbal"
          type="storm32_node.py"
          output="screen"
          respawn="true">
      <param name="frame_id" value="gimbal_ref" />
      <param name="port" value="/dev/gimbal" />
    </node>
  </group>
</launch>
